README.md

```diff
@@ -86,6 +86,7 @@ COOKIE_SECRET | The shared secret used in validating/calculating the session coo
 INTERNAL_API_URL | Internal API URL | `http://api.url`
 LANDING_PAGE_URL | Register OE landing Page | `/register-an-overseas-entity/sold-land-filter`
 UPDATE_LANDING_PAGE_URL | Update OE landing Page | `/update-an-overseas-entity/overseas-entity-query`
+REMOVE_LANDING_PAGE_URL | Remove OE landing Page | `/gov-uk-remove-guidance-page OR http://gov.uk.ch.site/roe-remove`
```

Comment:
i think this is meant to be the url to start the journey in test envs, so perhaps the 'choose resume / new " page?

Reply:
Changed this and also added some start URLs in the 'Running locally on Docker env' section above.